po/nb_NO.UTF-8.po

```diff
@@ -37,17 +38,19 @@ msgstr "OK"
 
 #: src/apprt/gtk/ui/1.5/config-errors-dialog.blp:5
 msgid "Configuration Errors"
-msgstr ""
+msgstr "Konfigurasjonsfeil"
 
 #: src/apprt/gtk/ui/1.5/config-errors-dialog.blp:6
 msgid ""
 "One or more configuration errors were found. Please review the errors below, "
 "and either reload your configuration or ignore these errors."
 msgstr ""
+"Én eller flere konfigurasjonsfeil ble funnet. Vennligst gjennomgå feilene under, "
```

Comment:
Just since I'm quite the language nerd and mentioned accents. This word means the number 1, while `en` is an article. So here it means "one or more", instead of "a or more".

Reply:
You god damn nerd😂But you are right.